string type
float64 a
float64 b
---
int8 done
